src/cli/analytics.rs

```diff
@@ -164,11 +167,60 @@ fn run(
             inner: vec![],
         })?;
 
+        if with_meta {
+            let mut query_results = vec![];
+            while let Some(mut query_result) = rt
+                .block_on(async { json_streamer.read_row().await })
+                .map_err(|e| ShellError::GenericError {
+                    error: format!("failed to read analytics query result: {}", e),
+                    msg: "".to_string(),
+                    span: None,
+                    help: None,
+                    inner: vec![],
+                })?
+            {
+                if !query_results.is_empty() {
+                    query_results.append(&mut vec![b','])
```

Comment:
Why is this needed?

Reply:
So that the results are formatted as a list